.travis.yml

```diff
@@ -1,12 +1,23 @@
+dist: focal
```

Comment:
I needed to upgrade to Ubuntu 20.04 because of:
```
unning redis.embedded.SpringDataConnectivityTest
/tmp/redis-18355853549604213145/redis-server-2.8.19-linux-aarch64: /lib/aarch64-linux-gnu/libm.so.6: version `GLIBC_2.29' not found (required by /tmp/redis-18355853549604213145/redis-server-2.8.19-linux-aarch64)
/tmp/redis-18355853549604213145/redis-server-2.8.19-linux-aarch64: /lib/aarch64-linux-gnu/libc.so.6: version `GLIBC_2.28' not found (required by /tmp/redis-18355853549604213145/redis-server-2.8.19-linux-aarch64)
```

This is because we used `arm64v8/ubuntu:20.04` to build the binary. To make it friendlier you may rebuild the ARM64 binary by using `arm64v8/ubuntu:18.04`, or even 16.04